rotkehlchen/exchanges/binance.py

```diff
@@ -170,20 +157,34 @@ def trade_from_binance(
 def create_binance_symbols_to_pair(exchange_data: Dict[str, Any]) -> Dict[str, BinancePair]:
     """Parses the result of 'exchangeInfo' endpoint and creates the symbols_to_pair mapping
     """
-    symbols_to_pair: Dict[str, BinancePair] = {}
+    result: Dict[str, BinancePair] = {}
     for symbol in exchange_data['symbols']:
         symbol_str = symbol['symbol']
         if isinstance(symbol_str, FVal):
             # the to_int here may rase but should never due to the if check above
             symbol_str = str(symbol_str.to_int(exact=True))
 
-        symbols_to_pair[symbol_str] = BinancePair(
+        result[symbol_str] = BinancePair(
             symbol=symbol_str,
             binance_base_asset=symbol['baseAsset'],
             binance_quote_asset=symbol['quoteAsset'],
         )
 
-    return symbols_to_pair
+    return result
+
+
+def query_binance_exchange_pairs() -> Dict[str, BinancePair]:
+    global symbols_to_pair
```

Comment:
no, no, no, no, no ... and ... no!

You should think a bit about what you want to achieve and what's the best way to go about doing it.

So what you want to achieve is ... be able to query the binance pairs without having the `Binance` object instantiated. The method you chose is not optimal because you are:

1. hard to understand
2. uses a global variable!!?!!?
3. Queries binance every time the pairs are needed

A better way would be:

1. Create a super simple table in the global DB that will store pair name, base, quote asset
2. Add setting in the global DB that remembers last time binance pairs were queried. If it's less than X hours (say 1 day? .. maybe even more?) just return the pairs in the DB
3. Otherwise query binance, cache the pairs in the DB and finally return the pairs in the DB
save it in the global DB after first